profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_dgt_connector/README.md

```diff
@@ -1,71 +1,70 @@
 TMGMT DGT Connector
 ===================
 This module integrates the [TMGMT module](https://www.drupal.org/project/tmgmt) 
-and the European Commission DGT connector services, enabling translation managers,
+and the European Commission DGT connector services, providing translation managers,
 a custom cart and all required logic.
 
-# Usage
-
-After enabling this module, it will be possible to translate:
-
-#### Content with length above 300 characters
-
-This will work like when using only the  ```TMGMT Poetry``` module and translator ```DGT Connector```.
+# Translatable strings
 
-#### Other translatable items using Small Jobs Cart
+The TMGMT DGT Connector module allows the translation of all default string types
+supported by the original TMGMT module. This includes:
+* Nodes
+* Blocks
+* Beans
+* Views
+* Locale strings
+* Field instances
+* Files
+* Menu and menu links
+* Vocabularies and Taxonomy terms
 
-It is now possible to bundle other different translatable items
-in the ```Small Jobs Cart```, using the translator ```TMGMT DGT Translator```:
+The module supports strings of all sizes but Nodes whose length is bigger
+than 300 characters have to be sent on individual requests.
 
-* Content with length under 300 characters;
-* Menu and Menu items;
-* Taxonomies and Terms;
-* Blocks 
+Other strings can and should be bundled together into single requests.
+Even though any string can be bundled together, for consistency sake it is recommended
+that related strings be bundled together (all items of a single menu for example).
 
-To add one translatable item to the ``Small Jobs Cart``, 
-go to the translation page of that item,
-chose the target languages and then use the "Send to cart" button.
 
-Then, in the ```Small Jobs Cart (admin/tmgmt/dgt_cart)```, you can send a bundle to be translated.
-Each bundle is a set of items having the same target languages set.
-Before sending a bundle for translation, please edit the bundle, then revise and complete
-the context information: Context URL should be filled and also any other
-hints that may help the person in charge of translation to understand the
-context where the item will be displayed.
+# Usage
 
+Requesting translations for content whose length is bigger than 300 follows the usual workflow
```

Comment:
Confusing and "whose" is used more when referring to a person. Suggestion:
"Requesting translations for content with length bigger than 300 characters"